app/helpers/template_helpers.py

```diff
@@ -14,215 +14,299 @@
-    template = f"{template.lower()}.html"
-    theme = cookie_session.get("theme", DEFAULT_THEME)
-    survey_title = lazy_gettext("Census 2021")
-    language_code = get_locale().language
-    page_header_context = get_page_header_context(language_code, theme)
-    page_header_context.update({"title": survey_title})
-    google_tag_manager_context = get_google_tag_manager_context()
-    cdn_url = f'{current_app.config["CDN_URL"]}{current_app.config["CDN_ASSETS_PATH"]}'
-    base_url = get_census_base_url(theme, language_code)
-    include_csrf_token = request.url_rule and "POST" in request.url_rule.methods
-    account_service_url = cookie_session.get(
-        "account_service_url", f"{CENSUS_EN_BASE_URL}en/start"
+
+@dataclass
+class CensusNISRAContextOptions(CensusContextOptions):
+    page_header_logo: str = "nisra-logo-en"
+    page_header_logo_alt: str = lazy_gettext(
+        "Northern Ireland Statistics and Research Agency logo"
     )
-    sign_out_url = url_for("session.get_sign_out")
-    static_content_urls = get_static_content_urls(language_code, base_url, theme)
-    footer_context = get_footer_context(
-        language_code,
-        static_content_urls,
-        sign_out_url,
-        theme,
+    header_logo: str = "nisra"
+    mobile_logo: str = "nisra-logo-en-mobile"
+    copyright_declaration: str = lazy_gettext(
+        "Crown copyright and database rights 2021 NIMA MOU577.501."
+    )
+    copyright_text: str = lazy_gettext(
+        "Use of address data is subject to the terms and conditions."
     )
+    footer_links: Iterable[Link] = field(
+        default_factory=lambda: [
+            Link(
+                lazy_gettext("Help"),
+                "help/help-with-the-questions/online-questions-help/",
+            ),
+            Link(lazy_gettext("Contact us"), "contact-us/"),
+        ]
+    )
+    footer_legal_links: Iterable[Link] = field(
+        default_factory=lambda: [
+            Link(lazy_gettext("Cookies"), "cookies/"),
+            Link(
+                lazy_gettext("Accessibility statement"),
+                "accessibility-statement/",
+            ),
+            Link(
+                lazy_gettext("Privacy and data protection"),
+                "privacy-and-data-protection/",
+            ),
+            Link(
+                lazy_gettext("Terms and conditions"),
+                "terms-and-conditions/",
+            ),
+        ]
+    )
+    powered_by_logo: str = "nisra-logo-black-en"
+    powered_by_logo_alt: str = "NISRA - Northern Ireland Statistics and Research Agency"
+
+
+def generate_context(
+    theme: str,
+    base_url: str,
+    language: str,
+) -> Dict[str, Any]:
+    context_options = {
+        "business": ContextOptions,
+        "health": ContextOptions,
+        "social": ContextOptions if language == "en" else ContextOptionsCymraeg,
+        "census": (
+            CensusContextOptions if language == "en" else CensusContextOptionsCymraeg
+        ),
```

Comment:
The logic here should be reversed as the default should always be `en`.

Reply:
Done